app/controllers/admin/config/fonts_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+class Admin::Config::FontsController < Admin::AdminController
+  def update
+    previous_default_text_size = SiteSetting.default_text_size
```

Comment:
Is this used? 🤔

Reply:
Good catch, thank you :) Yeah, this is leftover - I should remove it when I moved to `backfill` option in the SiteSetting::Update built by you